libs/wingsdk/src/shared/misc.ts

```diff
@@ -29,9 +29,9 @@ export function normalPath(path: string) {
 export async function runCommand(
   cmd: string,
   args: string[],
-  options?: ExecFileOptions
+  options?: ExecOptions
 ): Promise<any> {
-  const { stdout } = await execFilePromise(cmd, args, options);
+  const { stdout } = await execPromise(cmd + " " + args.join(" "), options);
```

Comment:
execFile and exec have different behaviors and performance characteristics so I don't think we should change runCommand to use it. I'll acknowledge `runCommand` is probably a bad name for something that isn't running in a shell...

Could we make two different versions of this function, one that uses a shell and one that doesn't? Ideally we would avoid the shell version whenever possible.

Reply:
Sure thing - I refactored it to a separate function